### PR TITLE
Remove "RECEIVE" button #796

### LINKED ISSUE #796: Remove "RECEIVE" button
Remove "RECEIVE" button, instead:
a -  Show address to the right of the amount (with copy Button)
b -  have a QR code button next to copy button, that pops up QR modal. 
*see attached image for Binance explorer as example*

image

image

### PR DESCRIPTION
**Pull request by thatStrangeGuyThorchain**
- extracted `QrCode` component to the separate one; added stories
- refactored `QrCode` logic to avoid multiple mounted elements
- added copybale address and modal with qr code to the asset info
- removed previous Receive stuff

closes #796

### CODE REVIEW

File: src/renderer/components/uielements/qrCode/qrCode.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import * as O from 'fp-ts/Option'
+import QRCode from 'qrcode'
+import * as ReactDom from 'react-dom'
+import { useIntl } from 'react-intl'
+
+import * as Styled from './qrCode.styles'
+
+type Props = {
+  text: O.Option<string>
+  noDataError: string
```
Comment (veado) [inline src/renderer/components/uielements/qrCode/qrCode.tsx:16]: Why is `text` an `Option` and not just a required prop? By required, no handling for `noDataError` is needed anymore.

File: src/renderer/components/wallet/assets/AssetDetails.tsx [hunk 2]
```diff
@@ -131,7 +122,7 @@ export const AssetDetails: React.FC<Props> = (props): JSX.Element => {
     [oRuneBnbBalance]
   )
 
-  const actionColSpanDesktop = isRuneBnbAsset || isRuneNativeAsset ? 8 : 12
+  const actionColSpanDesktop = isRuneBnbAsset || isRuneNativeAsset ? 12 : 12
```
Comment (veado) [inline src/renderer/components/wallet/assets/AssetDetails.tsx:125]:
```
 const actionColSpanDesktop = 12
```

File: src/renderer/components/uielements/assets/assetInfo/AssetInfo.tsx [hunk 3]
```diff
@@ -67,8 +80,41 @@ export const AssetInfo: React.FC<Props> = (props): JSX.Element => {
...
+        O.getOrElse(() => <></>)
+      ),
+    [walletInfo, setShowQrModal, oAsset]
+  )
+
+  const closeQrModal = useCallback(() => setShowQrModal(false), [setShowQrModal])
+
   return (
     <Styled.Card bordered={false} bodyStyle={{ display: 'flex', flexDirection: 'row' }}>
+      {showQrModal && (
```
Comment (veado) [inline src/renderer/components/uielements/assets/assetInfo/AssetInfo.tsx:117]: Don't show Modal if we don't have an address.

File: src/renderer/components/uielements/assets/assetInfo/AssetInfo.style.tsx [hunk 4]
```diff
@@ -32,27 +43,58 @@ export const CoinSubtitle = styled.p`
   text-transform: uppercase;
 `
 
-export const CoinPrice = styled.p`
+export const InfoContainer = styled('div')`
```
Comment (veado) [inline src/renderer/components/uielements/assets/assetInfo/AssetInfo.style.tsx:46]: [curious] Any reason to use `styled('div')` and not `styled.div`?

**Review by veado — APPROVED** [review]
Nice one! Just few questions.

File: src/renderer/components/uielements/assets/assetInfo/AssetInfo.style.tsx (hunk 4 above)
Comment (thatStrangeGuyThorchain) [reply src/renderer/components/uielements/assets/assetInfo/AssetInfo.style.tsx:46]: yep. you are right. no reason for that

**Review by veado — APPROVED** [review]

**Pull request merged**